pkg/registry/core/pod/storage/eviction_test.go

```diff
@@ -227,7 +227,7 @@ func TestEvictionDryRun(t *testing.T) {
 	for _, tc := range testcases {
 		t.Run(tc.name, func(t *testing.T) {
 			testContext := genericapirequest.WithNamespace(genericapirequest.NewContext(), metav1.NamespaceDefault)
-			storage, server := newFailDeleteUpdateStorage(t)
+			storage, _, _, server := newStorage(t)
```

Comment:
I don't know the difference between these two storage, do you happen to know what this means?

Reply:
The error I cited above was due to storage.NewKeyNotFoundError() being used by the newFailDeleteUpdateStorage.

Since we want the test to pass, normal storage should be used.